src/MainWindow.cpp

```diff
@@ -1507,6 +1552,19 @@ bool MainWindow::unsavedTailoringChanges() const
     return mUI.tailoringFileComboBox->currentIndex() == idx;
 }
 
+void MainWindow::fileChanged(const QString& path)
+{
+    if (path == mFSLastSeen)
+        return;
+    mFSLastSeen = path;
+
+    QMessageBox::information(
+        this, QObject::tr("SCAP Workbench"),
+        QObject::tr("Opened file was modified after opening: %1\n\n"
+                    "To reload, click File -> Reload Opened Content").arg(path)
+    );
+}
+
```

Comment:
At this moment, you show an info dialog. I suggest showing a dialog that would contain `Reload` and `Dismiss/Cancel` buttons, so users can reload right away.